widgetssdk/src/main/java/com/glia/widgets/launcher/EngagementLauncher.kt

```diff
@@ -80,115 +78,141 @@ interface EngagementLauncher {
-            endEngagementUseCase()
-            destroyChatController()
-        }
+    /**
+     * @return true if there is an ongoing engagement or enqueueing for any media type
+     */
+    private val isEngagementOrQueueing: Boolean
+        get() = isQueueingOrLiveEngagementUseCase.isQueueing || isQueueingOrLiveEngagementUseCase.hasOngoingLiveEngagement
 
-        when {
-            engagementTypeUseCase.isCallVisualizer -> callVisualizerController.showAlreadyInCvSnackBar()
-            isQueueingOrLiveEngagementUseCase.hasOngoingLiveEngagement || isQueueingOrLiveEngagementUseCase.isQueueingForLiveChat ->
-                activityLauncher.launchChat(context, Intention.RETURN_TO_CHAT)
+    /**
+     * @return true if there is an ongoing engagement or enqueueing for audio or video media type
+     */
+    private val isMediaEngagementOrMediaQueueing: Boolean
+        get() = engagementTypeUseCase.isMediaEngagement || isQueueingOrLiveEngagementUseCase.isQueueingForMedia
 
+    /**
+     * @return true if there is an ongoing engagement or enqueueing for live chat
+     */
+    private val isChatEngagementOrChatQueueing: Boolean
+        get() = engagementTypeUseCase.isChatEngagement || isQueueingOrLiveEngagementUseCase.isQueueingForLiveChat
+
+    override fun startChat(context: Context) {
+        when {
+            engagementTypeUseCase.isCallVisualizer -> snackbarController.showSnackBar(R.string.entry_widget_call_visualizer_description)
+            isMediaEngagementOrMediaQueueing -> snackbarController.showSnackBar(R.string.entry_widget_call_visualizer_description)
+            isEngagementOrQueueing -> activityLauncher.launchChat(context, Intention.RETURN_TO_CHAT)
             else -> hasOngoingSecureConversationUseCase(
-                onHasOngoingSecureConversation = { activityLauncher.launchChat(context, Intention.SC_DIALOG_ENQUEUE_FOR_TEXT) },
-                onNoOngoingSecureConversation = { activityLauncher.launchChat(context, Intention.LIVE_CHAT) }
+                onHasOngoingSecureConversation = {
+                    activityLauncher.launchChat(
+                        context,
+                        Intention.SC_DIALOG_ENQUEUE_FOR_TEXT
+                    )
+                },
+                onNoOngoingSecureConversation = {
+                    activityLauncher.launchChat(
+                        context,
+                        Intention.LIVE_CHAT
+                    )
+                }
             )
         }
     }
 
     override fun startChat(context: Context, visitorContextAssetId: String) {
         configurationManager.setVisitorContextAssetId(visitorContextAssetId)
-
         startChat(context)
     }
 
     override fun startAudioCall(context: Context) {
-        if (isQueueingOrLiveEngagementUseCase.isQueueingForLiveChat || isQueueingOrLiveEngagementUseCase.isQueueingForVideo) {
-            Logger.i(TAG, "Canceling ongoing queue ticket to create a new one")
```

Comment:
This log and a similar one in `fun startChat()` & `fun startVideoCall()` methods have been completely removed. Is this intended? If yes then please check whether [this doc](https://docs.google.com/spreadsheets/d/1ty9bhrbX3VTXBThI5f4GFzy5X-RbEHOCPWfXd7nGzbo/edit?gid=0#gid=0) needs updating.

Reply:
Good point. Thank you for a good catch. Yes, it's intended. I have asked the iOS team to confirm that they also don't have this log now. Will remove it from the doc once they confirm.